appveyor.yml

```diff
@@ -1,6 +1,6 @@
-version: 2.0.0-{build}
+version: 2.0.{build}
 pull_requests:
-  do_not_increment_build_number: true
+  do_not_increment_build_number: false

```

Comment:
Why the change?

Reply:
Needed to generate the correct package format for testing in VS. Going to revert to this before merging.